src/process-events.c

```diff
@@ -502,17 +485,18 @@ int BPF_KPROBE_SYSCALL(kprobe__sys_execve_4_11,
 
     // create the event early so it can be re-used to save stack space
     telemetry_event_t sev = {0};
-    ptelemetry_event_t ev = &sev;
 
-    u64 id = enter_exec_4_11(SP_EXECVE, ctx, ppid, luid, ev);
+    u64 id = enter_exec_4_11(SP_EXECVE, ctx, ppid, luid, &sev);
 
     if (!filename)
         goto Next;
 
-    __builtin_memset(ev, 0, sizeof(telemetry_event_t));
-    u64 off = 0;
-    char br = 0;
-    READ_VALUE_N(ev, TE_EXEC_FILENAME, filename, 5);
+    sev.id = id;
+    sev.telemetry_type = TE_EXEC_FILENAME;
+    sev.u.v.value[0] = '\0';
+    sev.u.v.truncated = FALSE;
+
+    push_string(ctx, &sev, filename);
```

Comment:
Same here as the previous comment. Since it passed, we should be good. If we're watching for verification errors we should be alerted to any edge cases we support where this might fail.